### ISSUE TITLE
Refactor examples

### ISSUE THREAD

**Issue by cjbassi** [gizak/termui#215]
- demo should demonstrate all widgets
- widget examples should demonstrate all functionality

**Comment by keithknott26** [comment]
Would it be possible to include the use of a theme in your example? I see that termui.UseTheme is no longer available in v2 and above.

**Comment by cjbassi** [comment]
Sure, that could be added.

The wiki should have an explanation of how the theme works now too. But the basic idea is that the theme allows you to define global colors that are inherited by widgets that you create. So you should be able do something like
```go
ui.Theme.Block.Title.Fg = ui.ColorBlue
```
and the rest of the widgets you create should default to having a blue title.

**Comment by DannyBen** [comment]
I agree with @keithknott26 - it would be nice to have an example with this, and a mention in the [Theming section of the Wiki page](https://github.com/gizak/termui/wiki#theming)
